src/app/components/Tokens/TokenTransfers.tsx

```diff
@@ -204,7 +204,7 @@ export const TokenTransfers: FC<TokenTransfersProps> = ({
   return (
     <Table
       columns={tableColumns}
-      rows={tableRows}
+      rows={isLoading ? undefined : tableRows}
```

Comment:
How come this is needed? we already pass `isLoading={isLoading}`

Reply:
Looks like wrong level of responsibility

Reply:
> How come this is needed? we already pass `isLoading={isLoading}`

Yes, but the [current implementation](https://github.com/oasisprotocol/explorer/blob/4fa627013e05c55f9a96036fba742d00332d2eed/src/app/components/Table/index.tsx#L128) of `Table` only considers `isLoadin` if there are no rows:

``` TypeScript
{!rows && isLoading && (
  <SkeletonTableRows rowsNumber={rowsNumber} columnsNumber={columns.length} />
)}
```

Reply:
> Looks like wrong level of responsibility

Fair point. The problem is that if we change the behavior of Table (to inverse the priorities, and make `isLoading` trump the passed rows), that might change the behavior of other pages in unpredictable ways.

However we could make this a configurable, opt-in behavior of Table, and only enable it for this page (for now), and then the code would be at the correct level, and we wouldn't break other pages.

I think I will do this now.

Reply:
Done.

Reply:
We should untangle that. Does any table ever rely on `!rows`? Origin seems to be https://github.com/oasisprotocol/explorer/pull/15

Reply:
I think the idea seems to be that if we already have content, and are currently reloading, there is no reason to hide the existing data; that would just cause flickering. Instead, keep the old data while loading, and then update when we have the new data. It's a valid use-case, and I think we are actually using it for updating tables, like the latest transactions and latest blocks. (And then we also have the code to highlight new rows.)

So yes, I think we _are_ using that.